lib/zipline/zip_generator.rb

```diff
@@ -59,8 +59,10 @@ def normalize(file)
       end
     end
 
-    def write_file(streamer, file, name)
-      streamer.write_deflated_file(name) do |writer_for_file|
+    def write_file(streamer, file, name, mtime)
+      mtime = mtime.nil? ? Time.now.utc : mtime
+
+      streamer.write_deflated_file(name, modification_time: mtime) do |writer_for_file|
```

Comment:
instead of having mtime, could you just have an optional "options" argument and pass it along untouched?

Reply:
You mean having `options = {}` and passing that along?